backend/src/controllers/auth.ts

```diff
@@ -8,9 +8,7 @@ import User from "../models/user";
 import sendEmail from "../utils/email";
 import { ReturnResponse } from "../utils/interfaces";
 import Mailgen from 'mailgen';
-
-
-import OTP from "../models/otp"
+import OTP from "../models/OTP";
```

Comment:
Why did you change a file name from lower case to upper case, the file name should be in lower case( if it is not a class) or at least it should be consistent in the whole project